### PR TITLE
mwprocapture: 1.2.4177 -> 1.3.0.4236

### LINKED ISSUE #122999: mwprocapture: Broken package?
I'm trying to use the mwprocapture nixos module (which uses the mwprocapture kernel package) on 20.09 (linuxPackages_latest), and I get the following:
```
Package ‘mwprocapture-1.2.4177-5.8.18’ in /nix/store/smd91bacvzcqbqldb58j47lqvgjxjwfs-nixos-20.09.1889.58f9c4c7d3a/nixos/pkgs/os-specific/linux/mwprocapture/default.nix:62 is marked as broken, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.
```
Is this known and/or already fixed somewhere? (Also, is it safe to use it anyways?)

I'd love to try and fix it further myself, but I don't know how to splice in a local/development nix file into my `boot.kernelPackages` for development purposes :sweat_smile: 

@MP2E (also CC @pwetzel)

### PR DESCRIPTION
**Pull request by pwetzel**
###### Motivation for this change
Driver version bump, needed for compatibility with kernel versions 5.8 and later. Fixes #122999.

This is needed by 21.05 also, if the committer is willing to cherry pick instead of waiting for my follow-up PR.
ZHF: #122042

Tested on MW ProCapture HDMI card.

###### Things done
- [x] Tested using sandboxing](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/mwprocapture/default.nix [hunk 1]
```diff
@@ -14,12 +11,12 @@ let
...
+  name = "mwprocapture-1.3.0.${version}-${kernel.version}";
+  version = "4236";
 
   src = fetchurl {
     url = "http://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
-    sha256 = "1nf51w9yixpvr767k49sfdb9n9rv5qc72f5yki1mkghbmabw7vys";
+    sha256 = "1mfgj84km276sq5i8dny1vqp2ycqpvgplrmpbqwnk230d0w3qs74";
   };
 
   nativeBuildInputs = [ kernel.moduleBuildDependencies ];
```
Comment (jonringer) [inline pkgs/os-specific/linux/mwprocapture/default.nix:22]: not related to changes
```suggestion
  nativeBuildInputs = kernel.moduleBuildDependencies;
```
it's already a list
```
nix-repl> :l
Added 14203 variables.

nix-repl> linuxPackages.kernel.moduleBuildDependencies
[ «derivation /nix/store/nn99rfl37w94yn8ripdmlh4ar3iixvmn-libelf-0.8.13.drv» ]
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/os-specific/linux/mwprocapture/default.nix (hunk 1 above)
Comment (pwetzel) [reply pkgs/os-specific/linux/mwprocapture/default.nix]: Thanks!

File: pkgs/os-specific/linux/mwprocapture/default.nix [hunk 2]
```diff
@@ -14,15 +11,15 @@ let
 
 in
 stdenv.mkDerivation rec {
-  name = "mwprocapture-1.2.${version}-${kernel.version}";
-  version = "4177";
+  name = "mwprocapture-1.3.0.${version}-${kernel.version}";
+  version = "4236";
 
   src = fetchurl {
     url = "http://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/mwprocapture/default.nix:18]:
```suggestion
    url = "https://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
```

Comment (pwetzel) [reply pkgs/os-specific/linux/mwprocapture/default.nix:18]: I think that https here doesn't give us any advantages? [Debian mirrors](https://www.debian.org/CD/http-ftp/#mirrors) rarely provide https, and the signing in use there is less robust than a pre-known hash like we have. https may instead be an added risk for failed builds if the source has certificate or other configuration problems.

While I prefer http, if that is best practice or convention on NixOS I don't mind changing it. I looked at the [manual](https://nixos.org/manual/nixpkgs/stable/#chap-pkgs-fetchers) and the example there gives http, for what it's worth.

Comment (SuperSandro2000) [reply pkgs/os-specific/linux/mwprocapture/default.nix:18]: Just use https if there is https. The contents of Debian mirrors are fully signed or anyone could do whatever with them.

> https may instead be an added risk for failed builds if the source has certificate or other configuration problems.

There is always a risk that the download somehow doesn't work. If that happens we can think about it at the time.

Comment (pwetzel) [reply pkgs/os-specific/linux/mwprocapture/default.nix]: Very well, thanks.

**Pull request merged**